lib/active_shipping/carriers/usps.rb

```diff
@@ -371,12 +371,15 @@ def build_us_rate_request(packages, origin_zip, destination_zip, options = {})
     #
     # package.options[:mail_type] -- one of [:package, :postcard, :matter_for_the_blind, :envelope].
     #                                 Defaults to :package.
-    def build_world_rate_request(packages, destination, options)
+    def build_world_rate_request(origin, packages, destination, options)
       country = COUNTRY_NAME_CONVERSIONS[destination.country.code(:alpha2).value] || destination.country.name
       xml_builder = Nokogiri::XML::Builder.new do |xml|
         xml.IntlRateV2Request('USERID' => @options[:login]) do
           Array(packages).each_with_index do |package, id|
             xml.Package('ID' => id) do
+              if country == "Canada"
```

Comment:
I bet everything works fine if you don't use an if statement, and that also makes it more robust to USPS adding additional countries to this requirement.